pallets/ajuna-awesome-avatars/src/tests.rs

```diff
@@ -615,6 +615,61 @@ mod season {
 			);
 		});
 	}
+
+	#[test]
+	fn active_season_hooks_should_update_active_season_on_block_initialize() {
+		ExtBuilder::default().organizer(ALICE).build().execute_with(|| {
+			assert!(AwesomeAvatars::active_season_id().is_none());
+
+			assert_ok!(AwesomeAvatars::new_season(
+				Origin::signed(ALICE),
+				Season {
+					early_start: 1,
+					start: 5,
+					end: 10,
+					max_mints: 1,
+					max_mythical_mints: 1,
+					rarity_tiers: get_rarity_tiers(),
+					max_variations: 1
+				}
+			));
+			assert_ok!(AwesomeAvatars::new_season(
+				Origin::signed(ALICE),
+				Season {
+					early_start: 11,
+					start: 15,
+					end: 20,
+					max_mints: 1,
+					max_mythical_mints: 1,
+					rarity_tiers: get_rarity_tiers(),
+					max_variations: 1
+				}
+			));
+
+			run_to_block(2);
+			assert!(AwesomeAvatars::active_season_id().is_some());
+			assert_eq!(AwesomeAvatars::active_season_id().unwrap(), 1);
```

Comment:
Let's simplify these lines into:
```suggestion
			assert_eq!(AwesomeAvatars::active_season_id(), Some(1));
```

Reply:
Changed!